bpf/network_events_monitoring.h

```diff
@@ -113,19 +107,12 @@ static inline int trace_network_events(struct sk_buff *skb, struct rh_psample_me
     }
 
     // there is no matching flows so lets create new one and add the network event metadata
-    u64 current_time = bpf_ktime_get_ns();
-    id.direction = INGRESS;
-    flow_metrics new_flow = {
-        .packets = 1,
-        .bytes = len,
-        .start_mono_time_ts = current_time,
-        .end_mono_time_ts = current_time,
-        .flags = flags,
+    additional_metrics new_flow = {
         .network_events_idx = 0,
     };
```

Comment:
there are instances where pkt hit the hook but not the tcx in that case u will see flows with o bytes and pkts did u check for that ?

Reply:
what sort of instances? My problem with that is it would duplicate counts, if this hook is called before the tc - or if the TC flow was just flushed - hence leading to over-estimate. I would rather keep strict responsibility boundaries : TC for counters, other hooks for additional features, and not mix up roles.

Reply:
then u need to make sure u have no rows in the UI with no counters as that is misleading